authprox/proto.go

```diff
@@ -0,0 +1,77 @@
+package authprox
+
+import (
+	"github.com/dedis/kyber"
+)
+
+// PROTOSTART
+// package authprox;
+
+// EnrollRequest is the request sent to this service to enroll
+// a user, authenticated by a certain type of external authentication.
+type EnrollRequest struct {
+	Type         string
+	Issuer       string
+	Secret       kyber.Scalar
```

Comment:
Since it's still possible to run conodes without TLS, would it be better to encrypt the secret using the public key of the conode? Or print a warning if the authproxy services are running without TLS.

Reply:
Given it is a one-time administrative operation, could we just document that it should be done across localhost? I know this is a very lazy solution, but I'm not sure this risk is one worth complicating the code for.

I could make apadmin refuse to send if !(tls || localhost).

Reply:
I'm fine with either, as long as it's clear to the user, either from the docs or the code.